iwyu_tool.py

```diff
@@ -89,6 +89,30 @@ def clang_formatter(output):
 }
 
 
+if sys.platform.startswith('win'):
+    # Case-insensitive match on Windows
+    def normcase(s):
+        return s.lower()
+else:
+    def normcase(s):
+        return s
+
+
+def is_msvc_driver(compile_command):
+    """ Return True if compile_command matches an MSVC CL-style driver. """
+    compile_command = normcase(compile_command)
+
+    if compile_command.endswith('cl.exe'):
+        # Native MSVC compiler or clang-cl.exe
+        return True
+
+    if compile_command.endswith('clang-cl'):
+        # Cross clang-cl on non-Windows
+        return True
+
+    return False
```

Comment:
It can get simpler by changing it to `return compile_command.endswith(('cl.exe', 'clang-cl'))`

Reply:
Neat! I didn't know `endswith` accepted a tuple. But that form would make it hard to place the explanatory comments, I think I prefer the current form (which explains how cl.exe also matches clang-cl.exe, and why clang-cl without extension is relevant).